### DISCUSSION TITLE
how-to/template-cheatsheet

### DISCUSSION THREAD

**Discussion by giscus** [aidenlx/obsidian-zotero#271] [Announcements]
# how-to/template-cheatsheet

https://zotlit.aidenlx.top/how-to/template-cheatsheet

**Comment by andreastheodor** [comment]
Hi. I'm really not sure what I'm doing wrong, but I can't seem to create new Literature Notes/import new highlights anymore. For some reason I was able to do it yesterday, albeit still inconsistently, but today I can't seem to at all. 

I follow the steps from above, in my Zotero library right click on an item, Obsidian Actions, Create Literature Notes. I do get a prompt "Allow this site to open the obsidian link with Obsidian?", click on "open link", then it redirects me to Obsidian but nothing happens.

Could this be a Zotero 7 beta issue? Or do you have any other idea why this might be happening?

**Comment by code-sur** [comment]
Add a property to front matter yml only if it exists. Compact (one-liner) version:
```yml
<%= it.prop ? `prop: "${it.prop}"` : undefined %>
```

**Comment by jenmfigueroa** [comment]
Hi! I'm interested in Grouping by annotation color, customized label -- however, I've copied and pasted this code but nothing happens. Is there something I'm missing?

**Reply by kelreeeeey** [reply]
where's your code??
